smart_card_connector_app/src/chrome-api-provider.js

```diff
@@ -408,12 +408,21 @@ class ChromeEventListener extends goog.Disposable {
               `${nameForLogs}(${argsDebug}): [${resultDebug}]`);
       reportResultFunction(requestId, ...result);
     };
-    this.chromeEvent_.addListener(this.callback_);
+    try {
+      this.chromeEvent_.addListener(this.callback_);
+    } catch (error) {
+      this.callback_ = null;
+      goog.log.warning(
+          logger, `Failed to subscribe to the event ${nameForLogs}: ${error}`);
+    }
   }
 
   /** @override */
   disposeInternal() {
-    this.chromeEvent_.removeListener(this.callback_);
+    // If subscribed to chrome event, unsubscribe.
+    if (this.callback_) {
+      this.chromeEvent_.removeListener(this.callback_);
+    }
     this.callback_ = null;
```

Comment:
nit: This line can also be moved inside `if`.

Reply:
done